Source/Fuse.Elements/Resources/ImageBackgroundLoad.uno

```diff
@@ -0,0 +1,213 @@
+using Uno;
+using Uno.Graphics;
+using Uno.UX;
+using Uno.IO;
+using Uno.Collections;
+using Uno.Compiler.ExportTargetInterop;
+using Uno.Graphics.Utils;
+using Fuse.Resources.Exif;
+
+namespace Fuse.Resources
+{
+    internal class ImageBackgroundLoad
+	{
+        Action<texture2D, byte[], ImageOrientation> _done;
+        Action<Exception> _fail;
+        Exception _exception;
+        ImageOrientation _orientation;
+        texture2D _tex;
+        byte[] _bytes;
+        int2 _targetSize;
+        string _filename;
+		bool _diskCache = false;
```

Comment:
This file has problems with mixed indentation.

Reply:
Oh sorry, my editor messing around. fixed with the last commit